hugo/content/research/2019/dora-report/index.md

```diff
@@ -9,4 +9,26 @@ tab_title: "DORA Report"
 ---
 
 ## Download the 2019 DORA Report
-[![Accelerate State of DevOps Report 2019](2019-dora-accelerate-state-of-devops-report.png)](2019-dora-accelerate-state-of-devops-report.pdf)
\ No newline at end of file
+[![Accelerate State of DevOps Report 2019](2019-dora-accelerate-state-of-devops-report.png)](2019-dora-accelerate-state-of-devops-report.pdf)
+
+
+The report emphasizes the importance of continuous delivery (CD) as a key technical practice that reduces the risk and cost of software releases. It also highlights the need for clear change management processes, recommending a shift from formal heavyweight change approvals to peer review-based approval during the development process, complemented by automation to detect, prevent, and correct potential issues early in the delivery lifecycle. Lastly, the report finds that a culture of psychological safety, trust, and respect positively contributes to both productivity and SDO performance.
+
+Key findings include:
+
+* Delivering software quickly, reliably, and safely is crucial for technology transformation and organizational performance, with elite performers being twice as likely to meet or exceed their performance goals.
+* Cloud continues to be a differentiator for elite performers and drives high performance, with elite performers 24 times more likely than low performers to utilize all five essential cloud computing capabilities.
+* Productivity can drive improvements in work/life balance and reductions in burnout, and organizations can support it by investing in tooling, information search, reducing technical debt, and fostering a culture of psychological safety.
+* Scaling DevOps effectively requires structural solutions that build communities, with high performers favoring strategies such as Communities of Practice and Grassroots initiatives.
+
+
+
+* Elite performers excel: The proportion of elite performers has almost tripled, highlighting that excellence in software delivery and organizational performance is achievable through adopting key capabilities.
+* Speed and stability go hand-in-hand: High performers achieve both speed and stability in software delivery, refuting the notion that there's an inherent trade-off between the two.
+* Cloud computing is a differentiator: Elite performance is closely linked to effective cloud adoption, with elite performers 24 times more likely to utilize all five essential cloud computing capabilities.
```

Comment:
This is basically a repeat of line 20; can we delete?

Reply:
Resolved in https://github.com/dora-team/dora.dev/pull/716/commits/4e363b72f5f63ed60a5ff15b6fcf8fd577050c49